odxtools/field.py

```diff
@@ -66,9 +66,8 @@ def structure(self) -> BasicStructure:
         """may be a Structure or a env-data-desc"""
         return odxrequire(self._structure, "EnvironmentDataDescription is not supported")
 
-    @property
-    def bit_length(self):
-        return self.structure.bit_length
+    def get_static_bit_length(self) -> Optional[int]:
```

Comment:
sure about this change?

Reply:
pretty much: to me, `.bit_length` gives the impression that it is just some data from the XML (which it sometimes indeed is) and not something which usual is calculated or not available at all. I'm open for better solutions/names, though.

Reply:
I am asking about returning None instead of `self.structure.bit_length`

Reply:
this is the base class for all fields, and as far as I can see, a static size can only be specified for `STATIC-FIELD` if it uses a structure with a static size. (Dynamically sized fields have a -- ehem -- dynamic length and also, `Structure.bit_length` does not exist anymore after this patch.) Since this is the base class for all fields, we should use the same approach as e.g. `DopBase` and return `None` whilst the derived classes override it if it makes sense...